product_docs/docs/biganimal/release/administering_cluster/notifications.mdx

```diff
@@ -0,0 +1,59 @@
+---
+title: Notifications 
+---
+
+BigAnimal supports receiving in-app and email notifications and allows you to choose receive notifications of a selected type. 
+
+Different types of events are sent as notifications. These notifications are set at different levels and users with different roles can configure this notifications. This table provides the list of events sent as notifications grouped by different levels at which they can be set:
```

Comment:
`These notifications are set at different levels` what do we mean here ?

Reply:
Different levels means Organization level and Project level

Reply:
could you clarify this in the text ? :pray: